src/plugins/intel_cpu/src/nodes/executors/dnnl/dnnl_fullyconnected_primitive.cpp

```diff
@@ -51,6 +51,7 @@ size_t DnnlFCPrimitive::Key::hash() const {
     seed = hash_combine(seed, get_attr_hash(*attr.get()));
     seed = hash_combine(seed, sparseWeights);
     seed = hash_combine(seed, transposedWeights);
+    seed = hash_combine(seed, isLLM);
```

Comment:
We don't need to update the hash since it already contains weights precision

Reply:
`DnnlFCPrimitive`'s constructor creates primitive `createPrimitiveDesc()` only based on information from `key`,   and to determined if actual weight dtype should be f16 (wdt) or f32(indt), we need `modelType` hint, so I have to add additional field here, I changed it to modelType.